lib/matplotlib/cbook/__init__.py

```diff
@@ -2242,10 +2242,23 @@ def type_name(tp):
                     type_name(type(v))))
 
 
-def _check_in_list(_values, **kwargs):
+def _check_in_list(_values, *, _print_supported_values=True, **kwargs):
     """
-    For each *key, value* pair in *kwargs*, check that *value* is in *_values*;
-    if not, raise an appropriate ValueError.
+    For each *key, value* pair in *kwargs*, check that *value* is in *_values*.
+
+    Parameters
+    ----------
+    _values : iterable
+        Sequence of values to check on
+    _print_supported_values : bool, default: True
+        Whether to print *_values* when raising ValueError
+    **kwargs : dict-like
```

Comment:
```suggestion
    **kwargs : dict
```
Let's keep it simple. Due to duck-typing any dict-like can stand in for a dict anyway.